src/decaylanguage/decay/decay.py

```diff
@@ -602,6 +602,33 @@ def ndecays(self) -> int:
         """
         return len(self.decays)
 
+    def to_string(self, arrow="->") -> str:
```

Comment:
Pre-commit complains that some type hints are missing. One of them is for sure here for `arrow`.